.env.example

```
@@ -2,37 +2,37 @@
 JWT_AUTH_DISABLED_MOCK_LOCAL_PRINCIPAL=
 
-# Development Settings
-SKIP_ETAG_VALIDATION=true
+# ITX Proxy Configuration (Required)
+# The service proxies all meeting requests to ITX Zoom API service
+ITX_BASE_URL=https://api.dev.itx.linuxfoundation.org
+# OAuth2 client ID for ITX
+ITX_CLIENT_ID=
+# OAuth2 client secret for ITX (keep secure!)
+ITX_CLIENT_SECRET=
+ITX_AUTH0_DOMAIN=linuxfoundation-dev.auth0.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ITX_AUTH0_DOMAIN key should go before the ITX_BASE_URL key

(UnorderedKey)